_posts/2019-02-15-introducing-aquery.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: posts
+title: Introducing Bazel Aquery
+authors:
+ - joeleba
+---
+
+_tl;dr: [`bazel aquery`](https://docs.bazel.build/versions/master/aquery.html) is a new bazel command that queries the action graph, and thus allows you to gain insights about the actions executed in a build (inputs, outputs, command line, …).
+`aquery`’s API is now stable and supported by the Bazel team._
```

Comment:
https://docs.bazel.build/versions/master/aquery.html says that the command/APi is experimental. Outdated?

Reply:
Indeed. Fixed.